### ISSUE TITLE
New Functionality: MAE split rule like in ExtraTreesRegressor

### ISSUE THREAD

**Issue by Polkas** [imbs-hl/ranger#545]
New feature development.

Do you think of implementing MAE split rule in contrast to regular MSE (variance) it is much better when we want to e.g. protection against outlier influence.

Notice: ExtraTreesRegressor sklearn solution is very inefficient. HistGradientBoostingRegressor offers more efficent solution - loss='least_absolute_deviation' although without variable importance option.

source: https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.ExtraTreesRegressor.html

**Comment by cargomoose** [comment]
Seconded. Can we please have an MAE split rule.

**Comment by mnwright** [comment]
Let's continue in #368.
